common/retry.py

```diff
@@ -67,9 +67,10 @@ def handle_retry(num_try, exception=None):
                 sleep(get_delay(num_try, delay, backoff))
                 return True
 
-            logs.error('Retrying on %s failed with %s. Raise.',
-                       function_with_type,
-                       sys.exc_info()[1])
+            if log_retries:
```

Comment:
This is a good fix.
But I think it would be better to eliminate the loop/circular import entirely, what do you think?

Reply:
Thanks Jonathan, I agree. 

I am closing this PR as I created [another one](https://github.com/google/fuzzbench/pull/1969) trying to get rid of the cycle entirely